features/admin.validation.v1/pages/validation-config-edit.tsx

```diff
@@ -140,6 +139,11 @@ export const ValidationConfigEditPage: FunctionComponent<MyAccountSettingsEditPa
     const [ legacyPasswordPolicies, setLegacyPasswordPolicies ] = useState<ConnectorPropertyInterface[]>([]);
 
     const isReadOnly: boolean = !useRequiredScopes(featureConfig?.governanceConnectors?.scopes?.update);
+    const hasRuleBasedPasswordExpiryReadPermissions: boolean =
+        useRequiredScopes(featureConfig?.ruleBasedPasswordExpiry?.scopes?.read);
+    const isRuleBasedPasswordExpiryDisabled: boolean =
+        disabledFeatures?.includes(ValidationConfigConstants.FEATURE_DICTIONARY.get("RULE_BASED_PASSWORD_EXPIRY"))
+        || !hasRuleBasedPasswordExpiryReadPermissions;
```

Comment:
let's not associate read permissions with feature flags.

```
const isRuleBasedPasswordExpiryFeatureDisabled: boolean =
        disabledFeatures?.includes(ValidationConfigConstants.FEATURE_DICTIONARY.get("RULE_BASED_PASSWORD_EXPIRY"))
```

then in L614,

```
if(!isRuleBasedPasswordExpiryDisabled && hasRuleBasedPasswordExpiryReadPermissions) {
```

Reply:
There are six places where isRuleBasedPasswordExpiryFeatureDisabled is used. So checking this in a single place would reduce the redundancy right?

Reply:
In that case, let's keep it as it is.